plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsContainerOptionsMapper.groovy

```diff
@@ -0,0 +1,159 @@
+package nextflow.cloud.aws.batch
+
+import com.amazonaws.services.batch.model.ContainerProperties
+import com.amazonaws.services.batch.model.KeyValuePair
+import com.amazonaws.services.batch.model.LinuxParameters
+import com.amazonaws.services.batch.model.Tmpfs
+import com.amazonaws.services.batch.model.Ulimit
+import groovy.transform.CompileStatic
+import nextflow.util.CmdLineOptionMap
+
+/**
+ * Maps task container options to AWS container properties
+ *
+ * @see https://docs.docker.com/engine/reference/commandline/run/
+ * @see https://docs.aws.amazon.com/batch/latest/APIReference/API_ContainerProperties.html
+ *
+ * @author Manuele Simi <manuele.simi@gmail.com>
+ */
+@CompileStatic
+class AwsContainerOptionsMapper {
+
+    static ContainerProperties createContainerOpts(CmdLineOptionMap options) {
```

Comment:
Made it return the `ContainerProperties` object

Reply:
Looks more readable. Thanks.